src/ontology/fbbt-edit.obo

```diff
@@ -147428,6 +147429,16 @@ intersection_of: has_soma_location FBbt:00003624 ! adult brain
 property_value: http://purl.org/dc/terms/contributor https://orcid.org/0000-0002-6095-8718
 property_value: http://purl.org/dc/terms/date "2023-06-13T16:47:28Z" xsd:dateTime
 
+[Term]
+id: FBbt:00059247
+name: visual neuropil intrinsic neuron
+def: "A neuron that is fully contained within the optic lobes or the ocellar ganglia." [orcid:0000-0002-0587-9355]
```

Comment:
Is that ocellar "ganglion" or "ganglia"?

Reply:
Ganglia is the plural form. There’s one ganglion per ocellus, so three ganglia.